google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/project/CloudProject.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.project;
+
+import com.google.auto.value.AutoValue;
+
+/** GCP project and account. */
+@AutoValue
+public abstract class CloudProject {
+  public static CloudProject create(String projectName, String googleUsername) {
+    return new AutoValue_CloudProject(projectName, googleUsername);
+  }
+
+  abstract String projectName();
```

Comment:
These should be public, otherwise callers outside of this package cannot access them.

Reply:
fixed.